src/SpaceTime/angular/AngularVelocity.cs

```diff
@@ -48,47 +39,36 @@ public static AngularVelocity FromDegrees(float degrees)
         /// <summary>
         /// Returns the numeric value of the angular velocity in radians.
         /// </summary>
-        public float NumericValue { get { return this.value; } }
+        public float NumericValue => value;
 
         /// <summary>
         /// Returns the angular value of the angular velocity.
         /// </summary>
-        public Angle AngleValue { get { return Angle.FromRadians(this.value); } }
+        public Angle AngleValue => Angle.FromRadians(value);
 
         /// <summary>
         /// Returns an angular velocity with value 0.
         /// </summary>
-        public static AngularVelocity Zero { get { return new AngularVelocity(0); } }
+        public static AngularVelocity Zero => new AngularVelocity(0);
 
         #endregion
 
         #region methods
 
         #region equality and hashcode
 
-        public bool Equals(AngularVelocity other)
-        {
-            return this.value == other.value;
-        }
+        // ReSharper disable once CompareOfFloatsByEqualityOperator
+        public bool Equals(AngularVelocity other) => value == other.value;
```

Comment:
Should we check `Math.abs(value - other.value) < float.Epsilon`? Is it possible to "propagate" the float comparison warning to the level using this method, since in a way it is still comparing two floats.